src/app/Processor/ProcessorDetail/ProcessorDetail.test.tsx

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import { customRender, waitForI18n } from "@utils/testUtils";
+import ProcessorDetail from "./ProcessorDetail";
+import { SinkProcessor, SourceProcessor } from "../../../types/Processor";
+
+describe("ProcessorDetail component", () => {
+  it("should display sink processor information", async () => {
+    const comp = customRender(<ProcessorDetail processor={sinkProcessor} />);
+    await waitForI18n(comp);
+
+    expect(comp.getByText("Processor type")).toBeInTheDocument();
+
+    expect(comp.getByTestId("processor-type-label")).toHaveTextContent("Sink");
+
+    expect(comp.queryByText("Filters")).toBeInTheDocument();
+
+    const filters = comp.container.querySelector(
```

Comment:
could we please use rather a `filter(s)Component` instead of `filters`?

I think with such change the line
```
    expect(filters).toHaveTextContent(sinkProcessor.filters?.[0].key as string);
```

Will become more readable

Reply:
Done